### PR TITLE
Model

### PR DESCRIPTION
**Pull request by cmougan**
Basic visualization

### CODE REVIEW

**Review by sunayana — APPROVED** [review]
I have not checked each polygon manually. But the structure looks fine to me

File: apps/retrieve_images.py [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import argparse
 import logging
-import os
+import os, sys
+
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
```
[the remark this answers is not in the document]
Comment (cmougan) [reply apps/retrieve_images.py:5]: I threw the error, when running the file as you said.

[the remark this answers is not in the document]
Comment (sunayana) [reply apps/retrieve_images.py:5]: Don't understand, could you explain which error is this?

[the remark this answers is not in the document]
Comment (cmougan) [reply apps/retrieve_images.py:5]: (energy_harvest_trust) bash-3.2$ python apps/retrieve_images.py data/input/other/polygon18.json 
Traceback (most recent call last):
  File "/Users/cmougan/Desktop/energy_harvest_trust/apps/retrieve_images.py", line 4, in <module>
    import src.preprocess_sentinel1 as ps1
ModuleNotFoundError: No module named 'src'

**Pull request merged**